src/app/core/entity/model/entity.ts

```diff
@@ -185,19 +185,21 @@ export class Entity {
   /**
    * Returns an array of strings by which the entity can be searched.
    *
-   * By default the parts of the "name" property (split at spaces) is used if it is present.
+   * By default the parts of the string representation (toString) split at spaces is used if it is present.
    *
    * <b>Overwrite this method in subtypes if you want an entity type to be searchable by other properties.</b>
    */
   @DatabaseField() get searchIndices(): string[] {
-    let indices = [];
-
-    // default indices generated from "name" property
-    if (typeof this["name"] === "string") {
-      indices = this["name"].split(" ");
+    if (
+      this.getConstructor().toStringAttributes === Entity.toStringAttributes &&
```

Comment:
When is this case necessary? Somebody could put `entityId` in the `toStringAttributes` but I don't think that case really needs to be covered.

Reply:
I needed this because for `User` the entityId is the same as the name ... and then user entities didn't show up in search.

If you have a more solid idea to circumvent this, I'll be happy! :see_no_evil: